backtesting/backtesting.py

```diff
@@ -1221,101 +1236,172 @@ def maximize(stats: pd.Series, _key=maximize):
             raise TypeError('`maximize` must be str (a field of backtest.run() result '
                             'Series) or a function that accepts result Series '
                             'and returns a number; the higher the better')
-
         if constraint is None:
 
             def constraint(_):
                 return True
 
         elif not callable(constraint):
-            raise TypeError("`constraint` must be a function that accepts a dict "
+            raise TypeError("constraint` must be a function that accepts a dict "
```

Comment:
While there are many changes here made by your opinionated editor (and which I'd very much like to eventually see reverted/cleaned up), this is probably not one of them. :smile: